packages/api/bin/serve.js

```diff
@@ -250,7 +255,37 @@ async function serveDistributionApi(stackName = 'localrun', done) {
   return distributionApp.listen(port, done);
 }
 
+/**
+ * Removes all additional data from tables and repopulates with original data.
+ *
+ * @param {string} user - defaults to local user, testUser
+ * @param {string} stackName - defaults to local stack, localrun
+ */
+async function resetTables(user = 'testUser', stackName = 'localrun') {
+  if (inTestMode()) {
```

Comment:
Could we have this available outside of test mode as well? I think it would be really useful to have a way to clean up our stacks, which would be especially awesome for a demo.

Also, it seems like we should be deleting granules here as well?

Reply:
Do we need to reset the `executions` table as well?